core/vm/memory_table.go

```diff
@@ -115,11 +115,11 @@ func memoryLog(stack *Stack) (uint64, bool) {
 }
 
 func memoryETX(stack *Stack) (uint64, bool) {
-	x, overflow := calcMemSize64(stack.Back(7), stack.Back(8))
+	x, overflow := calcMemSize64(stack.Back(6), stack.Back(7))
 	if overflow {
 		return 0, true
 	}
-	y, overflow := calcMemSize64(stack.Back(9), stack.Back(10))
+	y, overflow := calcMemSize64(stack.Back(8), stack.Back(9))
```

Comment:
This functions looks to be modified in the previous commit as well. I am confused as to which is the right version of the code. Even the jump table ETX.

Reply:
I see what @gameofpointers is saying. Just squash/split the changes so that the only patches going into the PR are the patches you wish to keep. The intermediate stuff that you remove later shouldn't be in the PR

Reply:
Fixed